src/lib/AST/AnyBlock.hpp

```diff
@@ -1874,6 +1874,90 @@ class FriendBlock
 
 //------------------------------------------------
 
+class NamespaceAliasBlock
+    : public TopLevelBlock<AliasInfo>
+{
+public:
+    using TopLevelBlock::TopLevelBlock;
+
+    Error
+    parseRecord(
+        Record const& R,
+        unsigned ID,
+        llvm::StringRef Blob) override
+    {
+        switch(ID)
+        {
+        case NAMESPACE_ALIAS_SYMBOL:
+            return decodeRecord(R, I->AliasedSymbol, Blob);
+        default:
+            return TopLevelBlock::parseRecord(R, ID, Blob);
+        }
+    }
+
+    Error
+    readSubBlock(
+        unsigned ID) override
+    {
+        switch(ID)
+        {
+        case BI_NAME_INFO_ID:
+        {
+            I->FullyQualifiedName = std::make_unique<NameInfo>();
```

Comment:
`NameInfoBlock` will allocate the `std::unique_ptr<NameInfo>`, so remove this line.